\documentclass[fleqn, 10pt]{manuscript}
%%\usepackage{setspace}
%%\doublespacing
\usepackage{soul, hyperref}
\usepackage[utf8]{inputenc}

\newcommand{\beginsupplement}{%
        \setcounter{table}{0}
        \renewcommand{\thetable}{S\arabic{table}}%
        \setcounter{figure}{0}
        \renewcommand{\thefigure}{S\arabic{figure}}%
     }

\title{A Structural and Functional Bioinformatics Study of QTY-designed Retinylidene Proteins}

\author[1]{Siqi Pan}
\affil[1]{Shanghai World Foreign Language Academy, 400 Baihua Street, Shanghai 200233, China}

\corrauthor[1]{Siqi Pan}{siqipan2008@outlook.com}

\keywords{bioinformatics; computational studies on functional properties of biological molecule; GPCRs; membrane protein structure; protein dynamics simulation}
\begin{abstract}

Retinylidene proteins are retinal-binding light-sensitive proteins found in organisms ranging from microbes to human. Microbial opsins have been utilized in optogenetics, while animal opsins are essential for vision and light-dependent metabolic functions. However, retinylidene proteins have hydrophobic transmembrane (TM) domains, which makes them challenging to study. In this structural and functional bioinformatics study, I use the QTY (Glutamine, Threonine, Tyrosine) code to design water-soluble QTY analogs of retinylidene proteins, including 9 human and 3 microbial opsins. I provide superpositions of the AlphaFold3-predicted hydrophobic native proteins and their water-soluble QTY analogs, and experimentally-determined structures when available. I also provide a comparison of surface hydrophobicity of the variants. Despite significant changes to the protein sequence (35.53\% to 50.24\% in the TM domain), protein characteristics and structures are well-preserved. Furthermore, I run molecular dynamics (MD) simulations of native and QTY-designed OPN2 (rhodopsin) and analyze their response to the isomerization of 11-cis-retinal to all-trans-retinal. The results show that the QTY analog has similar functional behavior to the native protein. The findings of this study indicate that the QTY code can be used as a robust tool to design water-solubility retinylidene proteins. These have potential applications in protein studies, therapeutic treatments, and bioengineering. 

\end{abstract}

\begin{document}

\flushbottom
\maketitle
\thispagestyle{empty}

\section*{Introduction}

Retinylidene proteins are photochemically reactive proteins that are bound to or can bind to retinal (vitamin A aldehyde) as their chromophore \citep{Spudich_2000}. For convenience, I will use the term ``opsin'' interchangeably with ``retinylidene protein'', despite the fact that ``opsin'' sometimes refers specifically to the chromophore-free apoprotein. Retinylidene proteins are divided into two groups, animal opsins and microbial opsins, which are evolutionarily distinct but share common characteristics such as having 7 transmembrane (TM) domains and a retinal-binding lysine residue \citep{Yee_2013, Spudich_2000}. In this study, I will consider 9 animal opsins and 3 microbial opsins. 

\textbf{Animal opsins} are a type of class A GPCR (G-protein coupled receptor), which are characterized by 7 transmembrane domains, the NPxxY motif, and activation of G-proteins through the outward movement of TM6. \citep{Zhou_2019}. Almost all animal opsins include a lysine residue that forms a Schiff base link with the retinal \citep{Guhmann_2022}. When retinal absorbs a photon, it isomerizes, usually changing from 11-cis to all-trans. The subsequent conformational changes of the protein are well studied using bovine rhodopsin, which changes from dark state to BATHO state, then LUMI, META I, and META II \citep{Okada_2001}. Proton transfer plays an important role in this process \citep{Mahalingam_2008}. Afterward, the protein bleaches and returns to the dark state, completing what is called the photocycle. As the protein conformation changes to META, TM6 moves characteristically outward, activating the G-protein. In photoreceptor cells, the G-protein, transducin, activates a phosphodiesterase, which hydrolyzes cGMP into GMP, decreasing the activity of cGMP-gated cation channels and hyperpolarizing the cell \citep{Chabre_1989}. 

In this study, I select 9 opsins expressed in the human nervous system: OPN1MW (UniProt ID: \href{https://www.uniprot.org/uniprotkb/P04001/entry}{P04001}), OPN1LW (UniProt ID: \href{https://www.uniprot.org/uniprotkb/P04000/entry}{P04001}), OPN1SW (UniProt ID: \href{https://www.uniprot.org/uniprotkb/P03999/entry}{P03999}), OPN2 (UniProt ID: \href{https://www.uniprot.org/uniprotkb/P08100/entry}{P04001}), OPN3 (UniProt ID: \href{https://www.uniprot.org/uniprotkb/Q9H1Y3/entry}{Q9H1Y3}), OPN4 (UniProt ID: \href{https://www.uniprot.org/uniprotkb/Q9UHM6/entry}{Q9UHM6}), OPN5 (UniProt ID: \href{https://www.uniprot.org/uniprotkb/Q6U736/entry}{Q6U736}), RGR (UniProt ID: \href{https://www.uniprot.org/uniprotkb/P47804/entry}{P47804}) and RRH (UniProt ID: \href{https://www.uniprot.org/uniprotkb/O14718/entry}{O14718}). They belong to several evolutionarily distinct families of animal opsins \citep{Terakita_2005}. 

\textbf{OPN1MW} (Medium-Wave-sensitive Opsin 1), \textbf{OPN1LW} (Long-Wave-sensitive Opsin 1) and \textbf{OPN1SW} (Short-Wave-sensitive Opsin 1) are expressed in retinal cone photoreceptors and are responsible for color vision \citep{Bowmaker_1980}. Certain variants of OPN1MW, OPN1LW and OPN1SW respectively cause deutanopia, protanopia, and tritanopia, which are different types of color blindness \citep{Ueyama_2002, Baraas_2012}. The absence of both functional OPN1MW and OPN1LW causes blue cone monochromacy, an X-linked congenital cone dysfunction syndrome \citep{Wissinger_2022}, and cone dystrophy 5, an X-linked cone dystrophy \citep{Gardner_2010}. 

\textbf{OPN2} (Opsin 2), also known as rhodopsin, is expressed in retinal rod photoreceptors and is responsible for vision at low light intensity \citep{Hubbard_1958}. Certain variants lead to autosomal recessive or autosomal dominant retinitis pigmentosa and congenital stationary night blindness \citep{Fanelli_2021}. OPN2 is a representative animal opsin, one of the first studied. In fact, bovine OPN2 is the first opsin to be sequenced \citep{Nathans_1984}, as well as the first GPCR whose crystal structure was resolved experimentally \citep{Palczeski_2000}. Many studies on the functional mechanisms of animal opsins also focus on OPN2. Consequently, I choose OPN2 to conduct a functional analysis in this study in order to further explore the effectiveness of the QTY code in redesigning retinylidene proteins. 

\textbf{OPN3} (Opsin 3), also known as encephalopsin or panopsin, is activated by blue and ultraviolet A light. It was discovered in the brain \citep{Blackshaw_1999}. It is also expressed in melanocytes and keratinocytes in the skin and regulates functions such as melanogenesis, cell differentiation, and glucose uptake. Its expression is also found in the liver, pancreas, kidney, lung, heart, and skeletal muscles. When expressed in neurons, the release of neurotransmitters is inhibited by light, making OPN3 a useful inhibitory optogenetic tool \citep{Copits_2021}. 

\textbf{OPN4} (Opsin 4), also known as melanopsin, is expressed in ipRGC (intrinsically photosensitive Retinal Ganglion Cells) in the ganglion cell layer in the retina \citep{Provencio_1998}. It is essential for non-image-forming responses to light, including the pupillary reflex, optokinetic visual tracking response, and photoentrainment and regulation of circadian rhythm. Certain variants of OPN4 lead to seasonal affective disorder and other circadian rhythm disorders \citep{Berson_2002}. Rendering OPN4-containing ipRGCs capable of image formation is also a potential pathway for the treatment of various eye diseases such as retinitis pigmentosa and diabetic retinopathy. 

\textbf{OPN5} (Opsin 5), also known as neuropsin, is activated by blue and ultraviolet A light \citep{Tarttelin_2003}. It is expressed in the retina and contributes to the regulation of light-dependent vascular development and photoentrainment in the cornea and retina \citep{Buhr_2015}. Certain variants of OPN5 may lead to cycloplegia, paralysis of the ciliary muscle in the eye. 

\textbf{RGR} (RPE-retinal GPCR) is expressed in RPE (Retinal Pigmented Epithelium) and M\"uller cells in the retina \citep{Shen_1994}. Unlike the aforementioned human opsins, RGR preferentially binds all-trans-retinal and may catalyze its isomerization into 11-cis-retinal via a retinochrome-like mechanism. It is expressed only in tissue surrounding photoreceptors and plays a role in the light-dependent synthesis of visual chromophore \citep{Radu_2008}.

\textbf{RRH} (RPE-derived Rhodopsin Homolog), also known as peropsin, is localized in the microvilli of RPE cells that surround photoreceptor outer segments \citep{Sun_1997}. It is another protein that preferentially binds to all-trans-retinal \citep{Cook_2017}. Although not much information is known about RRH, it can reasonably be inferred that it photoisomerizes all-trans-retinal to 11-cis-retinal and may play a role in the upkeep of photoreceptor functions. 

\textbf{Microbial opsins} are transmembrane ion pumps or channels \citep{Findlay_1986}. They are also 7TM, though this is due to convergent evolution rather than homology \citep{Yee_2013}. The chromophore, retinal, usually isomerizes from all-trans to 13-cis, different from the case of animal opsins \citep{Findlay_1986}. In addition, microbial opsins often form oligomers to carry out their functions \citep{Gmelin_2007}.

In this study, I select 3 microbial opsins: BACR (UniProt ID: \href{https://www.uniprot.org/uniprotkb/P02945/entry}{P02945}), BACH (UniProt ID: \href{https://www.uniprot.org/uniprotkb/B0R2U4/entry}{B0R2U4}), ChR2 (UniProt ID: \href{https://www.uniprot.org/uniprotkb/Q8RUT8/entry}{Q8RUT8}). 

\textbf{BACR} (Bacteriorhodopsin) is a light-driven proton pump. It is one of the first microbial opsins discovered \citep{Oesterhelt_1971}. \textbf{BACH} (Halorhodopsin) is a light-driven chloride pump activated by yellow light \citep{Schobert_1982}. \textbf{ChR2} (Channelrhodopsin 2) is a light-activated sodium channel activated by blue light \citep{Nagel_2003}. BACH and ChR2 are among the first optogenetic tools \citep{Zhang_2007, Han_2007}. BACH is used for inhibition, while ChR2 is used for excitation. 

I have asked whether retinylidene proteins could be redesigned to be more soluble. Retinylidene proteins are all integral membrane proteins with seven transmembrane alpha helices embedded in a lipid bilayer. Because of the hydrophobic properties of the transmembrane domains, they are not water-soluble without the aid of detergents. 

Bacteriorhodopsin has been the subject of several protein-solubilizing studies, though with limited success \citep{Qing_2022}. Recently, researchers leveraged a neural network, SolubleMPNN, which was built upon ProteinMPNN, to engineer soluble variants of bacteriorhodopsin while maintaining its ligand-binding ability and light-sensing function \citep{Nikolaev_2024}.

Instead of taking a computational approach, I apply the QTY (Glutamine, Threonine, Tyrosine) code to systematically engineer water-soluble analogs with reduced hydrophobicity in membrane proteins. There are structural similarities between hydrophobic and polar amino acids: leucine (L) and glutamine (Q); isoleucine (I) / valine (V) and threonine (T); and phenylalanine (F) and tyrosine (Y), as can be observed on high-resolution electron density maps \citep{Zhang_2018, Zhang_2022, Tegler_2020}. This justifies the replacement of hydrophobic amino acids with polar ones. Zhang et al. initially applied the QTY code to design several detergent-free chemokines receptors, all of which retained structural thermal stability and native ligand-binding and enzymatic activities despite substantial changes to the transmembrane domain \citep{Zhang_2018, Tegler_2020}. Zhang's team also applied the QTY code to design water-soluble GPCRs, including chemokine receptors \citep{Zhang_2018, Qing_2019, Tegler_2020, Skuhersky_2021}, cytokine receptors \citep{Hao_2020}, and olfactory receptors \citep{Skuhersky_2021, Johnsson_2025}. In addition to GPCR, they used the QTY code to design water-soluble glucose transporters \citep{Smorodina_2022}, ABC transporters \citep{Pan_2024}, monoamine neurotransmitter transporters \citep{Karagol_Karagol_Alper_2024}, glutamate transporters \citep{Karagol_Karagol_Smorodina_2024, Karagol_2024}, mitochondrial megacomplex \citep{Chen_2025}, antibodies \citep{Li_Wang_2024}, potassium channels \citep{Smorodina_2024}, receptor kinases \citep{Li_Tang_2024}, transmembrane enzymes \citep{Chen_Pan_2025}, as well as bacterial enzymes with beta barrel structures \citep{Sajeev-Sheeja_2023, Sajeev-Sheeja_2024}. The QTY-designed water-soluble CXCR4 chemokine receptor was successfully used in biomimetic sensors \citep{Qing_2023}. 

AlphaFold2 was released by Google DeepMind in July 2021 \citep{Jumper_2021}. It has greatly facilitated the study of QTY-designed protein variants. Subsequently, AlphaFold3 was released in May 2024, featuring improved architecture and improved efficiency \citep{Abramson_2024}. Furthermore, AlphaFold3 enabled the accurate prediction of complexes of multiple proteins, as well as complexes with modified residues, nucleic acids, ions, and certain ligands. QTY studies have made use of these new features \citep{Chen_2025, Johnsson_2025}. 

GROMACS is a molecular dynamics (MD) simulation program released in 1995 \citep{Berendsen_1995}. Its 5.0 version was released in 2015 \citep{Abraham_2015}. GROMACS enables an efficient and realistic simulation of biomolecular systems and has been used to investigate the structural and functional properties of QTY-designed protein variants in various studies \citep{Karagol_2024, Li_Tang_2024, Smorodina_2024, Li_Wang_2024, Johnsson_2025}. 

In this paper, I apply the QTY code to redesign 9 human opsins and 3 microbial opsins. I provide the superpositions of the AlphaFold3-predicted hydrophobic native proteins and their water-soluble QTY variants, and experimentally-determined structures when available. I also provide a comparison of the surface hydrophobicity of the variants. Furthermore, I run MD simulations of native and QTY-designed OPN2 and analyze their response to the 11-cis to all-trans isomerization of the retinal chromophore. 

\section*{Methods}

\subsection*{QTY design, protein sequence alignment and other characteristics}

The native protein sequences for OPN1MW, OPN1LW, OPN1SW, OPN2, OPN3, OPN4, OPN5, RGR, RRH, BACR, BACH, and ChR2 were obtained from UniProt (\url{https://www.uniprot.org}). The sequence for ChR2 was truncated according to the experimentally determined structure in the RCSB PDB (PDB ID: \href{https://www.rcsb.org/structure/8ZAN}{8ZAN}). The QTY designs were performed through the Protein Solubilizing Server (\url{https://pss.sjtu.edu.cn/}) \citep{Tao_2022}. For ChR2, the secondary structure was provided to the server in SS3 format according to the RCSB PDB structure. 

\subsection*{AlphaFold3 predictions}

I predicted the structures of native proteins and their QTY variants using the AlphaFold3 website (\url{https://alphafoldserver.com}) \citep{Abramson_2024}. For microbial opsins, the dimers and trimers have identical subunits, i.e., are homodimers and homotrimers. In AlphaFold3, this was achieved by altering the number of protein copies. 

\subsection*{Structure superpositions}

PDB files for native protein structures determined experimentally by X-ray diffraction or electron microscopy were taken from RCSB PDB: OPN2 (PDB ID: \href{https://www.rcsb.org/structure/5W0P}{5W0P}), BACR (PDB ID: \href{https://www.rcsb.org/structure/7XJC}{7XJC}), BACH (PDB ID: \href{https://www.rcsb.org/structure/2JAF}{2JAF}), ChR2 (PDB ID: \href{https://www.rcsb.org/structure/8ZAN}{8ZAN}). The AlphaFold3-predicted native and QTY variants were taken directly from the most probable predicted structure. Superpositions were performed via the command ``super'' in PyMOL (\url{https://pymol.org}). I removed unstructured loops at the N and C terminals for the sake of clarity. 

\subsection*{Structure visualization}

I used PyMOL (\url{https://pymol.org}) to superpose the native predicted protein structures, their QTY variants, and the experimentally determined structures for the proteins where these existed. I then used UCSF ChimeraX (\url{https://www.cgl.ucsf.edu/chimerax/}) to render each protein model with hydrophobicity patches.  

\subsection*{Molecular dynamics simulations}

All MD simulations and analyses were executed on a desktop computer with Intel Xeon Platinum 8352V Processor, 256 GB RAM, and 2 NVIDIA GPU (GeForce RTX 4090) with 24 GB VRAM each. All MD simulations were conducted using GROMACS 2024.5 \citep{Abraham_2015} with the CHARMM36m all-atom force field \citep{Huang_2017}. Data for retinal were obtained from NAMD Wiki (\url{https://www.ks.uiuc.edu/Research/namd/wiki/index.cgi?RetinalTop} for topology and \url{https://www.ks.uiuc.edu/Research/namd/wiki/index.cgi?RetinalPar} for parameters) and manually integrated into the protein file. Initial structures, configuration files, and command-line codes for the simulations are publicly available on Zenodo (\url{https://doi.org/10.5281/zenodo.15377505}). 

Regarding native OPN2, the protein-membrane system was constructed using the web-based membrane builder CHARMM-GUI and downloaded in GROMACS format \citep{Jo_2008, Wu_2014, Lee_2016}. The protein was centered in a rectangular box. The generated membrane models consisted of 40\% POPC, 40\% POPE, 10\% POPS, and 10\% cholesterol, which simulated a rod photoreceptor disk membrane where native OPN2 is usually expressed \citep{Albert_2005}. The system was solvated in TIP3P water with 150 mM NaCl. NaCl was used instead of KCl to simulate the environment of rod photoreceptor outer segments \citep{Govardovskii_1971}. 11-cis-retinal was manually added to the protein files. LINCS constraints were used for constraints, and Verlet integrator was used. Electrostatics was handled with Particle Mesh Ewald (PME), with both Coulomb and van der Waals interaction cutoffs set at 1.2 nm. The energy of the system was minimized using the steepest descent until the maximum forces converged below 1000 kJ/mol/nm. The standard six-step CHARMM-GUI NP$\gamma$T equilibration protocol \citep{Jo_2008} was used, with two 125-ps NVT equilibration simulations, and one 125-ps and three 500-ps NP$\gamma$T equilibration simulations. Temperature and pressure were maintained at 303.15 K and 1.0 bar, respectively, using the V-rescale thermostat and C-rescale barostat with surface-tension coupling. Following equilibration, a 10-ns production MD simulation was run. The retinal molecule was then manually switched to the all-trans state by rotating C13 to C15 and associated hydrogen and methyl groups by 180 degrees around the C11-C12 bond. This mimicked the effect of the absorption of a photon by 11-cis-retinal. Energy minimization and equilibration were repeated exactly as before, and then a 120-ns production MD simulation was run. 

Regarding QTY-designed OPN2, I found that the helix 9 of the QTY analog flung up and adhered to the surface that was originally the transmembrane domain, which disrupted the normal conformation. Therefore, this helix was truncated. The system was constructed directly using GROMACS. The protein was centered in a rectangular box with dimensions equal to those of native OPN2. The equilibration involved a sequence of simulations: one 250-ps NVT equilibration, followed by one 125-ps and one 1500-ps NPT equilibrations, totaling an equilibration duration that matches that of the natural OPN2. All other configurations and parameters were identical to those of the original OPN2.

After the simulations, I extracted frames from the trajectory to inspect the structures before and after retinal isomerization. I also calculated the RMSD (``gmx rms'' command), number of hydrogen bonds (``gmx hbond'' command) and water molecules (``gmx select'' command) inside the binding pocket, interaction energy (including short-range Coulombic interaction energy and short-range Lennard-Jones energy) (``gmx energy'' command), the RMSF (``gmx rmsf'' command), and radius of gyration (``gmx gyrate'' command). The detailed commands are publicly available in the online database for this study. 

\section*{Results and Discussion}

\subsection*{The rationale of the QTY code}

The transmembrane segments of membrane proteins are hydrophobic in nature. This leads to a challenge to study their structure and function. Traditionally, detergent must be used to purify them. Zhang et al. considered a different approach, i.e., via systematic soluble protein design. There exist structural similarities, as can be observed on high-resolution electron density maps, between certain hydrophobic and hydrophilic amino acids: leucine (L) and glutamine (Q), isoleucine (I) or valine (V) and threonine (T), and phenylalanine (F) with tyrosine (Y). This fact enables systematic replacement of L with Q, I/V with T, and F with Y in all transmembrane segments of the membrane protein, which Zhang et al. named the QTY code. QTY analogs are less hydrophobic than native membrane proteins. Although their amino acid sequences are significantly changed, they still exhibit relatively preserved structure, isoelectric points (pI), and molecular weights (MW) compared to the native membrane proteins (Table~\ref{tb:characteristics}). 

\subsection*{Protein sequence alignments and other characteristics of retinylidene proteins}

The protein sequences of 9 human opsins and 3 microbial opsins were aligned with their QTY analogs (Figure~\ref{fig:sequences}). The QTY substitution resulted in an overall variation of the sequence of 15.48\% to 30.04\% and a variation of transmembrane domain of 35.53\% to 50.24\%. Despite changes in amino acid composition and sequence, the pI only experienced a slight change, between 0.00 and 0.16. This is because the amino acids Q, T, and Y have neutral charges and do not introduce additional charges to the system.  The MW increased slightly by a value between 0.04kDa and 0.60kDa. This is because the hydrophilic amino acids usually have nitrogen or oxygen atoms in place of carbon atoms in hydrophobic acids and thus may have higher mass: L (131.17Da) \textit{vs} Q (146.14Da); I (131.17Da) and V (117.15Da) \textit{vs} T (119.12Da); F (165.19Da) \textit{vs} Y (181.19Da). 

\subsection*{Superpositions of AlphaFold3-predicted native human opsins, their water-soluble QTY variants, and experimentally determined structures}

Although the chromophore, retinal, is bound to retinylidene proteins in an internal pocket, it is only incorporated into the protein after protein folding is complete. In other words, protein folding does not depend on the chromophore. Thus, it is justified to predict the ligand-free forms of the proteins. I used AlphaFold3 to predict the structure of the 9 native human opsins and their water-soluble QTY analogs. The structures superposed very well (Figure~\ref{fig:humansup} \textbf{a)} to \textbf{i)}). For OPN2, which has an available X-ray diffraction structure, I also superposed the experimentally determined structure with the AlphaFold3-predicted structures. Overall, the root mean square distances (RMSD) were small, from 0.307{\AA} to 0.611\AA, with only one exception of OPN2$^{\textrm{QTY}}$ \textit{vs} OPN2$^{\textrm{EXP}}$ (RMSD = 0.999\AA). This shows that the water-soluble QTY analogs are quite similar to native proteins in terms of structure. 

All 9 of these human opsins belong to the same large family, i.e. animal opsins. However, they belong to different subfamilies and have structural differences from each other. In order to investigate the extent to which the water-soluble QTY analogs preserve these differences, I also conducted pairwise superpositions within all 9 native human opsins and within all 9 QTY analogs of human opsins (Figure~\ref{fig:humansup} \textbf{j)} and \textbf{k)}). As observed in the tables, there was a large degree of similarity between the RMSD between a pair of native proteins and that between the corresponding pair of QTY analogs. Therefore, I conclude that the structural differences are relatively well-preserved before and after the QTY substitution. However, whether the functional differences are preserved remains a problem to study. 

\subsection*{Superpositions of AlphaFold3-predicted native microbial opsins, their water-soluble QTY variants, and experimentally determined structures}

I used AlphaFold3 to predict the structure of monomers of the 3 native microbial opsins, their water-soluble QTY analogs. Since microbial opsins often have to form oligomers to be functional, I also predicted the structure of native and QTY variants of BACR trimers, BACH trimers, and ChR2 dimers. These predicted structures were also superposed with experimentally determined structures (X-ray diffraction or electron microscopy). The structures superposed very well (Figure~\ref{fig:microbialsup}). Overall, the root mean square differences (RMSD) were small, with the highest value being 0.685\AA. This shows that the water-soluble QTY analogs are likely capable of forming oligomers with structures similar to those of native proteins. 

\subsection*{Analysis of the hydrophobic surface of native retinylidene proteins and their water-soluble QTY variants}

As transmembrane proteins, retinylidene proteins require detergents to be separated from the lipid bilayer. The detergents act as an interface between the hydrophobic transmembrane domain of the protein and the surrounding water, thus solubilizing the protein. If these proteins are exposed in water without detergents, their hydrophobicity will induce them to aggregate and precipitate, with their structures and functions disrupted. 

The hydrophobic portions of the protein surface are represented in yellow (Figure~\ref{fig:hydrophobicity}). In the native proteins, there are large hydrophobic patches in the transmembrane domains, which are embedded within the lipid bilayer. The nonpolar and hydrophobic amino acids, including leucine (L), isoleucine (I), valine (V), phenylalanine (F), alanine (A), methionine (M), and tryptophan (W), interact with the hydrophobic hydrocarbon chains and expel water. 

I investigated the surface hydrophobicity of water-soluble QTY-analogs in comparison with the native proteins. After applying the QTY code to respectively replace the hydrophobic amino acids L, I/V, and F with hydrophilic amino acids glutamine (Q), threonine (T), and tyrosine (Y), the hydrophobic surface areas were significantly reduced. More importantly, since the electron density maps of the corresponding amino acids are similar, the alpha helices in QTY analogs had a similar contour to those in the native proteins. The QTY analogs successfully retained its structural integrity and stability. 

\subsection*{AlphaFold3 predictions}

DeepMind released AlphaFold3 in May 2024, marking a significant leap in accuracy for modeling across biomolecular space. This latest iteration outperforms state-of-the-art docking tools and its predecessor, AlphaFold-Multimer v.2.3, in protein structure and protein-protein interaction predictions \citep{Abramson_2024}. AlphaFold3 reduces the reliance on multiple sequence alignment by integrating a diffusion-based model, enabling it to predict a broader spectrum of biomolecules, including ligands, ions, nucleic acids, modified residues, and large protein megacomplexes. On October 9, 2024, DeepMind’s founders, Demis Hassabis and John Jumper, received the Nobel Prize in Chemistry for revolutionizing protein structure prediction through leveraging machine learning. 

AlphaFold3 is easily accessible online (\url{https://alphafoldserver.com}), allowing users to make 30 predictions a day at present. The structures of the QTY analogs were predicted using the AlphaFold3 server, which was free of charge, and the results were produced within a few minutes. 

DeepMind also collaborated with the EBI to make over 214 million predicted protein structures available through the AlphaFold Protein Structure Database (\url{https://alphafold.ebi.ac.uk}). This number is continuously expanding, with the quality of predictions further improving with the advent of AlphaFold 3. 

However, AlphaFold3 still has limitations that have been encountered in this study. Although AlphaFold3 allows the structural prediction of complexes that include small-molecule ligands, it only supports a small range of ligands, which does not include retinal. I was therefore unable to predict the structure of chromophore-bound states of retinylidene proteins with AlphaFold3. Consequently, I chose another approach, namely molecular dynamics simulation, in order to investigate the behavior of QTY opsin analogs when they are bound to the chromophore. 

\subsection*{Simulation of 11-cis to all-trans isomerization of retinal in native OPN2 and its QTY analog}

QTY substitutions do not introduce changes to the essential NPxxY motif or the retinal-binding lysine of human retinylidene proteins. Therefore, I proposed that QTY analogs of human opsins may have conserved functions and carried out a molecular dynamics (MD) simulation. I recorded and analyzed the behavior of native OPN2 and its QTY analog before and after the 11-cis to all-trans isomerization. Please note that my aim is not to simulate the whole photocycle of OPN2, which takes longer than 1ms \citep{Fanelli_2021}, but rather to find proof that the QTY analog conserves the function of being activated by isomerization of retinal. 

The MD simulation provided evidence of conformational change in both native and QTY analogs (Figure~\ref{fig:conformation}). The native protein experienced a slower change from its original conformation, while the QTY analog showed more fluctuations and sudden changes. Under further inspection, this phenomenon might be due to the entrance of water molecules into the QTY analog. In addition, retinal molecule had a greater degree of change in conformation in the native protein than in the QTY analog. Nevertheless, the retinal in the QTY analog was still able to cause changes to surrounding residues, sometimes doing so via intervening water molecules. It may be noted that the RMSD was still fluctuating, which means that the protein had not yet stabailized. This was expected, since the OPN2 was still in the process of transition from BATH to LUMI state, and this provided evidence for the activation of the protein in response to isomerization of retinal. 

Zooming in on the retinal-binding pocket, I observed that retinal is held within its binding pocket by several forces in both the native and QTY analog: the counterions of the retinal Schiff base (RSB) (113E and 181E), the hydrophobic interactions with neighboring residues (for example, 265W, 268Y, and the 187--189 beta strand), and, sometimes, steric collisions (Figure~\ref{fig:binding}). Since retinal is a ligand that binds in an internal pocket, I was unable to calculate the Gibbs free energy of its binding. Nevertheless, the distributions of interaction energies were found to be similar in native OPN2 and the water-soluble QTY analog, except for small differences: native OPN2 experienced a change in interaction energy at about $t=50$ns due to the entrance of water near 113E; the QTY analog had more negative interaction energy at residues 208 and 212 due to phenylalanine (F) to tyrosine (Y) substitutions. Apart from these, the similarity was a strong suggestion of functional conservation. Finally, I observed very different hydrogen bonding and water molecule patterns inside the binding pocket, which was mainly due to several F to Y substitutions in that region. This might cause retinal to interact with adjacent residues in a subtly different way, and might lead to a different absorption peak since the environment of the RSB, i.e., the binding pocket, determines the spectral characteristic of opsins \citep{Fenno_2011}. 

Overall, I observed that QTY-designed OPN2 had relatively similar behavior to the native variant despite certain small differences caused by decreased hydrophobicity. The function of OPN2$^{\textrm{QTY}}$ awaits to be verified by experiments. 

\subsection*{Future scopes and potential applications}

The findings of this study indicate that the QTY code could be used as a robust tool to design water-soluble retinylidene proteins. 

This could, in the first place, facilitate the study of these proteins. The water-solubility of QTY analogs makes it easier to purify and investigate proteins, especially those such as RGR and RRH, which have not been studied much but may have considerable clinical relevance. The water-solubility also reduces the difficulty in recording the different phases of the photocycle, in comparison with native membrane opsins. Another application may be the rapid design of new optogenetic tools. 

Furthermore, water-soluble QTY opsins could be useful clinically, since they could be delivered and/or expressed in the eye without forming aggregates and precipitating. It is not unreasonable to hypothesize that QTY opsins can still pass through the photocycle and interact with downstream signaling proteins. In light of this, QTY opsins may facilitate the optogenetic therapy of ophthalmological diseases \citep{Sakai_2022}, providing a potential pathway in restoring basic vision for those who have lost it. 

Last but not least, water-soluble opsins have the potential to be harvested in mass and may be used to design new biomimetic light-sensing systems, which may have applications in bioengineering. 

\subsection*{Conclusion}

In this study, I selected 12 retinylidene proteins, including 9 human opsins (OPN1MW, OPN1LW, OPN1SW, OPN2, OPN3, OPN4, OPN5, RGR, and RRH) and 3 microbial opsins (BACR, BACH, and ChR2), that had clinical implications and various potential applications. I applied the QTY code to convert the hydrophobic transmembrane alpha-helices into hydrophilic ones and thus create water-soluble QTY analogs of the proteins. Then, I used AlphaFold3 to predict the structures of the native proteins as well as the QTY analogs, superposed them, and found that the structure of the proteins were well conserved despite substantial residue substitutions. I also inspected the surface hydrophobicity of the proteins and found a great decrease in hydrophobic patches in the transmembrane surface. Next, I chose a representative retinylidene protein, OPN2, and employed molecular dynamics simulations to investigate the behavior of native and QTY analog of OPN2. I found that the QTY analog was capable of conformational change and effective protein-ligand interaction. These findings revealed that QTY-designed OPN2, and retinylidene proteins in general, might have conserved structure and function. I believe that these water-soluble QTY variants may have potentials in research, therapeutic treatments, and bioengineering. 


\section*{Supplementary Information} 

The Supplementary Information is available at...

\section*{Data Availability Statement} 

The data for the AlphaFold3-predicted structures and MD simulation setting files and commands can be found on Zenodo (\url{https://doi.org/10.5281/zenodo.15377505}). 

\section*{Author contributions}

Conceptualization: S.P.; Formal analysis: S.P.; Investigation: Methodology: S.P.; Validation: S.P.; Data curation: S.P.; Writing—original draft preparation: S.P.; Review and editing: S.P. 

\section*{Acknowledgements}

S.P. thanks Prof. Shuguang Zhang of the Lab of Molecular Architecture, Media Lab, Massachusetts Institute of Technology for inspiring the topic of this study as well as providing valuable advice on research methods and manuscript preparation. 

\section*{Financial Support}

There is no financial support for this digital bioinformatics study. I only used free tools that are publicly available. 

\section*{Competing Interests}

Massachusetts Institute of Technology (MIT) filed several patent applications for the QTY code for GPCRs, which do not include the human opsins. There are no competing interests.


\section*{Ethics Statement}

All methods were carried out in accordance with relevant guidelines and regulations. All experimental protocols were approved by a named institutional and licensing committee. Neither human biological samples nor human subjects were used in the study. This is a completely digital structural bioinformatic study using the publicly available AlphaFold3 machine learning program and GROMACS molecular dynamics simulation program.


\bibliography{references}

\begin{table}[htbp]
	\centering
	\caption{\textbf{Protein characteristics of 12 retinylidene proteins and their QTY analogs. } Blue: human opsins; purple: microbial opsins. RMSD: root mean square distance; pI: isoelectric focusing; MW: molecular weight; TM: transmembrane. }
	\label{tb:characteristics}
	\includegraphics[width=\linewidth]{Figures/characteristics.jpg}
\end{table}


\begin{figure}[htbp]
	\centering
	\includegraphics[width=\linewidth]{Figures/sequences.jpg}
	\caption{\textbf{The protein sequence alignments of 12 retinylidene proteins and their QTY analogs. } Blue: human opsins; purple: microbial opsins. The symbols $|$ and $*$ indicate that amino acids are identical or different, respectively. Amino acids L, I/V, and F in TM (transmembrane) alpha helices (shown in blue above the sequences) are replaced with Q, T, and Y respectively. The variation in the TM domain ranges from 35.53\% to 50.24\% while the overall variation rate ranges from 15.48\% to 30.04\%. The characteristics of the proteins are shown above the sequences. Despite large variation rates, the pI only experiences a slight change between 0.00 and 0.16 and the MW increases slightly by a value between 0.04kDa and 0.60kDa. The enlarged individual sequence alignments are available in Supplementary Information. 
    The alignments are 
    \textbf{a)} OPN1MW \textit{vs} OPN1MW$^{\textrm{QTY}}$, 
    \textbf{b)} OPN1LW \textit{vs} OPN1LW$^{\textrm{QTY}}$, 
    \textbf{c)} OPN1SW \textit{vs} OPN1SW$^{\textrm{QTY}}$, 
    \textbf{d)} OPN2 \textit{vs} OPN2$^{\textrm{QTY}}$, 
    \textbf{e)} OPN3 \textit{vs} OPN3$^{\textrm{QTY}}$, 
    \textbf{f)} OPN4 \textit{vs} OPN4$^{\textrm{QTY}}$, 
    \textbf{g)} OPN5 \textit{vs} OPN5$^{\textrm{QTY}}$, 
    \textbf{h)} RGR \textit{vs} RGR$^{\textrm{QTY}}$, 
    \textbf{i)} RRH \textit{vs} RRH$^{\textrm{QTY}}$, 
    \textbf{j)} BACR \textit{vs} BACR$^{\textrm{QTY}}$, 
    \textbf{k)} BACH \textit{vs} BACH$^{\textrm{QTY}}$, 
    \textbf{l)} ChR2 \textit{vs} ChR2$^{\textrm{QTY}}$. }
	\label{fig:sequences}
\end{figure}

\begin{figure}[htbp]
	\centering
	\includegraphics[width=\linewidth]{Figures/superposition-human-pairwise.jpg}
	\caption{\textbf{Superposition of AlphaFold3-predicted native human retinylidene proteins, their QTY analogs, and experimentally determined structures. } For clarity, unstructured N- and C- terminal ends are deleted. 
    For \textbf{a)} to \textbf{i)}, despite significant changes in the protein sequence, the structures superpose very well. The root mean square distance (RMSD) values are quite small, from 0.307{\AA} to 0.611\AA, with only one exception (OPN2$^{\textrm{QTY}}$ \textit{vs} OPN2$^{\textrm{EXP}}$, RMSD = 0.999\AA). Green: AlphaFold3-predicted native structure; cyan: AlphaFold3-predicted QTY analog structure; magenta: experimentally determined structure. The superpositions are 
    \textbf{a)} OPN1MW$^{\textrm{AF3}}$ \textit{vs} OPN1MW$^{\textrm{QTY}}$, 
    \textbf{b)} OPN1LW$^{\textrm{AF3}}$ \textit{vs} OPN1LW$^{\textrm{QTY}}$, 
    \textbf{c)} OPN1SW$^{\textrm{AF3}}$ \textit{vs} OPN1SW$^{\textrm{QTY}}$, 
    \textbf{d)} OPN2$^{\textrm{AF3}}$ \textit{vs} OPN2$^{\textrm{QTY}}$ \textit{vs} OPN2$^{\textrm{EXP}}$, 
    \textbf{e)} OPN3$^{\textrm{AF3}}$ \textit{vs} OPN3$^{\textrm{QTY}}$, 
    \textbf{f)} OPN4$^{\textrm{AF3}}$ \textit{vs} OPN4$^{\textrm{QTY}}$, 
    \textbf{g)} OPN5$^{\textrm{AF3}}$ \textit{vs} OPN5$^{\textrm{QTY}}$, 
    \textbf{h)} RGR$^{\textrm{AF3}}$ \textit{vs} RGR$^{\textrm{QTY}}$, 
    \textbf{i)} RRH$^{\textrm{AF3}}$ \textit{vs} RRH$^{\textrm{QTY}}$. 
    For \textbf{j)} and \textbf{k)}, there is a large degree of similarity between the RMSD between a pair of native proteins and that between the corresponding pair of QTY analogs. Green: OPN1MW; red: OPN1LW; blue: OPN1SW; purple: OPN2; cyan: OPN3; gray: OPN4; olive: OPN5; orange: RGR; pink: RRH. The superpositions are
    \textbf{j)} OPN1MW$^{\textrm{AF3}}$ \textit{vs} OPN1LW$^{\textrm{AF3}}$ \textit{vs} OPN1SW$^{\textrm{AF3}}$ \textit{vs} OPN2$^{\textrm{AF3}}$ \textit{vs} OPN3$^{\textrm{AF3}}$ \textit{vs} OPN4$^{\textrm{AF3}}$ \textit{vs} OPN5$^{\textrm{AF3}}$ \textit{vs} RGR$^{\textrm{AF3}}$ \textit{vs} RRH$^{\textrm{AF3}}$, 
    \textbf{k)} OPN1MW$^{\textrm{QTY}}$ \textit{vs} OPN1LW$^{\textrm{QTY}}$ \textit{vs} OPN1SW$^{\textrm{QTY}}$ \textit{vs} OPN2$^{\textrm{QTY}}$ \textit{vs} OPN3$^{\textrm{QTY}}$ \textit{vs} OPN4$^{\textrm{QTY}}$ \textit{vs} OPN5$^{\textrm{QTY}}$ \textit{vs} RGR$^{\textrm{QTY}}$ \textit{vs} RRH$^{\textrm{QTY}}$. 
    }
	\label{fig:humansup}
\end{figure}

\begin{figure}[htbp]
	\centering
	\includegraphics[width=\linewidth]{Figures/superposition-microbial.jpg}
	\caption{\textbf{Superposition of AlphaFold3-predicted native microbial retinylidene proteins, their QTY analogs, and experimentally determined structures. } Despite significant changes in the protein sequence, the structures superpose very well. The root mean square distance (RMSD) values are quite small, with the highest being 0.685\AA. For clarity, unstructured N- and C- terminal ends are deleted. 
    Green: AlphaFold3-predicted native structure; cyan: AlphaFold3-predicted QTY analog structure; magenta: experimentally determined structure. 
    The superpositions are 
    \textbf{a)} BACR$^{\textrm{AF3}}$ \textit{vs} BACR$^{\textrm{QTY}}$ \textit{vs} BACR$^{\textrm{EXP}}$ monomer, 
    \textbf{b)} BACR$^{\textrm{AF3}}$ \textit{vs} BACR$^{\textrm{QTY}}$ \textit{vs} BACR$^{\textrm{EXP}}$ trimer, 
    \textbf{c)} BACH$^{\textrm{AF3}}$ \textit{vs} BACH$^{\textrm{QTY}}$ \textit{vs} BACH$^{\textrm{EXP}}$ monomer, 
    \textbf{d)} BACH$^{\textrm{AF3}}$ \textit{vs} BACH$^{\textrm{QTY}}$ \textit{vs} BACH$^{\textrm{EXP}}$ trimer, 
    \textbf{e)} ChR2$^{\textrm{AF3}}$ \textit{vs} ChR2$^{\textrm{QTY}}$ \textit{vs} ChR2$^{\textrm{EXP}}$ monomer, 
    \textbf{f)} ChR2$^{\textrm{AF3}}$ \textit{vs} ChR2$^{\textrm{QTY}}$ \textit{vs} ChR2$^{\textrm{EXP}}$ dimer. }
	\label{fig:microbialsup}
\end{figure}

\begin{figure}[htbp]
	\centering
	\includegraphics[width=\linewidth]{Figures/hydrophobicity.jpg}
	\caption{\textbf{Hydrophobic surface of 12 retinylidene proteins and their water-soluble QTY analogs. } Hydrophobic patches are shown in yellow while hydrophilic patches are shown in cyan. The native proteins have many hydrophobic patches due to the presence of hydrophobic amino acids, including L, I, V, and F. After QTY substitution, hydrophilic Q, T, and Y have respectively replaced hydrophobic L, I/V, and F, and the hydrophobic patches in the surface of transmembrane helices have become more hydrophilic. In addition, the surface shape of the native and QTY analogs are very similar. For clarity, unstructured N- and C- terminal ends are deleted. 
    The comparisons are 
    \textbf{a)} OPN1MW \textit{vs} OPN1MW$^{\textrm{QTY}}$, 
    \textbf{b)} OPN1LW \textit{vs} OPN1LW$^{\textrm{QTY}}$, 
    \textbf{c)} OPN1SW \textit{vs} OPN1SW$^{\textrm{QTY}}$, 
    \textbf{d)} OPN2 \textit{vs} OPN2$^{\textrm{QTY}}$, 
    \textbf{e)} OPN3 \textit{vs} OPN3$^{\textrm{QTY}}$, 
    \textbf{f)} OPN4 \textit{vs} OPN4$^{\textrm{QTY}}$, 
    \textbf{g)} OPN5 \textit{vs} OPN5$^{\textrm{QTY}}$, 
    \textbf{h)} RGR \textit{vs} RGR$^{\textrm{QTY}}$, 
    \textbf{i)} RRH \textit{vs} RRH$^{\textrm{QTY}}$, 
    \textbf{j)} BACR \textit{vs} BACR$^{\textrm{QTY}}$ monomer, 
    \textbf{k)} BACH \textit{vs} BACH$^{\textrm{QTY}}$ monomer, 
    \textbf{l)} ChR2 \textit{vs} ChR2$^{\textrm{QTY}}$ monomer, 
    \textbf{m)} BACR \textit{vs} BACR$^{\textrm{QTY}}$ trimer, 
    \textbf{n)} BACH \textit{vs} BACH$^{\textrm{QTY}}$ trimer, 
    \textbf{o)} ChR2 \textit{vs} ChR2$^{\textrm{QTY}}$ dimer. 
    }
	\label{fig:hydrophobicity}
\end{figure}

\begin{figure}[htbp]
	\centering
	\includegraphics[width=\linewidth]{Figures/function-conformation.jpg}
	\caption{\textbf{The conformational changes of native OPN2 and its QTY analog before and after 11-cis to all-trans isomerization of the chromophore, retinal. } \textbf{a)} and \textbf{b)}: 1ns running averages of the root mean square distances (RMSD) of the protein-retinal complex, transmembrane helix 6 (TM6), the retinal-binding pocket, and retinal. By convention, the isomerization is set at time 0ns, which is indicated by a brown, vertical dashed line. \textbf{c)}: superpositions between cis-state OPN2, trans-state OPN2, cis-state OPN2$^{\textrm{QTY}}$, and trans-state OPN2$^{\textrm{QTY}}$. Both OPN2 and OPN2$^{\textrm{QTY}}$ exhibit conformational changes, with RMSDs greater than 2\AA. Blue: cis-state protein, orange: 11-cis-retinal; yellow: trans-state protein, greenish cyan: all-trans retinal. }
	\label{fig:conformation}
\end{figure}

\begin{figure}[htbp]
	\centering
	\includegraphics[width=\linewidth]{Figures/function-binding.jpg}
\end{figure}

\begin{figure}[htbp]
	\caption{\textbf{Changes in the retinal-binding pocket and protein-ligand interaction in native OPN2 and its QTY analog before and after 11-cis to all-trans isomerization of the chromophore. } \textbf{a)} and \textbf{b)}: close-ups of the binding pocket in cis-state. Protein-ligand interactions with lengths shorter or equal to 3.5\AA are shown in the figure. Blue: protein residues, orange: 11-cis-retinal; green dashed lines: ion bridge; yellow dashed lines: van der Waals and/or hydrophobic interactions. \textbf{c)} and \textbf{d)}: close-ups of the binding pocket in trans-state. Protein-ligand interactions with lengths shorter or equal to 3.5\AA are shown in the figure. Yellow: protein residues, greenish cyan: all-trans-retinal; green dashed lines: ion bridge; yellow dashed lines: van der Waals and/or hydrophobic interactions. \textbf{e)} and \textbf{f)}: the interaction energies (IE) between the protein, the binding pocket, individual residues, and retinal. IE is the sum of the short-range Coulombic interaction energy and short-range Lennard-Jones energy. Please note that IE is a product of MD simulation and is not necessarily a ``real'' physical quantity. For clarity, IE is rescaled using the signed pseudo logarithm ($y = \textrm{sign}(x)\cdot\ln(|x|+1)$). By convention, the isomerization is set at time 0ns, which is indicated by a brown, vertical dashed line. The large changes in IE in OPN2 around $t=50$ns are due to the entrance of water molecules into the binding pocket, near 113E. The QTY analog has more negative IE at residues 208 and 212 due to F to Y substitutions. Besides from these, the similarity between the IE of OPN2 and OPN2$^{\textrm{QTY}}$ is a strong suggestion of functional conservation. \textbf{g)} and \textbf{h)}: the number of hydrogen bonds formed between residues in the binding pocket, and the number of water molecules within the binding pocket. By convention, the isomerization is set at time 0ns, which is indicated by a brown, vertical dashed line. }
	\label{fig:binding}
\end{figure}

\end{document}